ccloudvm.go

```diff
@@ -233,7 +233,8 @@ func createFlags(ctx context.Context, ws *workspace) (*workload, bool, error) {
 	return wkl, debug, nil
 }
 
-func startFlags(in *VMSpec) error {
+func startFlags(in *VMSpec) (bool, error) {
```

Comment:
I sort of feel that it makes sense to include this option in the VMspec structure.  This would allow you to specify the debug port in the workloads but still override it on the command line.  Even better, your choice would be remembered, so the next time you started the VM you wouldn't need to specify the option.

Reply:
We could also show the debug port if specified in the status output of the status command, which would be nice.